js/hotgraphicView.js

```diff
@@ -0,0 +1,301 @@
+define([
+    'core/js/adapt',
+    'core/js/views/componentView'
+], function(Adapt, ComponentView) {
+
+    var HotGraphicView = ComponentView.extend({
+
+        initialize: function() {
+            this.listenTo(Adapt, 'remove', this.remove);
+            this.listenTo(this.model, 'change:_isVisible', this.toggleVisibility);
+            this.listenTo(Adapt, 'accessibility:toggle', this.onAccessibilityToggle);
+            
+            this.model.set('_globals', Adapt.course.get('_globals'));
```

Comment:
I _think_ we can probably drop this line and change line 231 to `Adapt.course.get('_globals')`